src/Compilers/CSharp/Portable/Syntax/LambdaUtilities.cs

```diff
@@ -47,6 +47,15 @@ public static bool IsLambda(SyntaxNode node)
         public static SyntaxNode GetLambda(SyntaxNode lambdaBody)
         {
             var lambda = lambdaBody.Parent;
+            if (lambda.Kind() == SyntaxKind.ArrowExpressionClause)
```

Comment:
That's not how the EnC analyzer represents lambdas though. We really need the body to be the expression.

---
In reply to: [134613176](https://github.com/dotnet/roslyn/pull/21599#discussion_r134613176) [](ancestors = 134613176)